python/adjoint/filters.py

```diff
@@ -7,6 +7,8 @@
 import meep as mp
 from scipy import special
 from scipy import signal
+import skfmm
```

Comment:
Is it necessary to include this additional module just for its signed distance function (rather than write our own or find a similar function in e.g. SciPy)? skfmm does not seem to be widely used which could make installing it a challenge for some users building from source.

Reply:
> could make installing it a challenge for some users building from source.

From their [docs](https://github.com/scikit-fmm/scikit-fmm):

```
pip install scikit-fmm
```

What's challenging about this particular package vs `numpy`, `scipy`, etc.?

It's convenient for testing, but we can remove it if it really is a challenge to install.

Reply:
I ran into some package conflicts during the `pip install`. In general, it is useful to keep the number of package dependencies, particularly smaller ones, to a minimum.